release-notes/DIS/DIS_2.39.md

```diff
@@ -0,0 +1,140 @@
+---
+uid: DIS_2.39
+---
+
+# DIS 2.39
+
+## New features
+
+### IDE
+
+#### Enhanced NuGet support [ID_33544] [ID_34113] [ID_34115] [ID_34117]
+
+Up to now, in QAction and EXE projects, DIS would add references to the DataMiner DLL files of the locally installed DataMiner Agent or, if no DataMiner Agent was installed on the local machine, to copies of those files shipped with the DIS installation package. Instead, DIS will now add a reference to a DataMiner DevPack, i.e. a NuGet package that contains the core DataMiner DLL files of a specific DataMiner version. As a separate DataMiner DevPack is available for every released DataMiner version, adapting a solution to a specific DataMiner version is now merely a question of making sure the solution links to the DevPack of that version.
+
+The DataMiner DevPacks can be found on the [official NuGet store](https://www.nuget.org/packages?q=skylinecommunications).
+
+When you open a protocol solution or an Automation script solution in which the QAction and EXE projects still contain references to separate DLL files instead of a DataMiner DevPack, a banner will appear, giving you the opportunity to replace those references by references to the required DevPacks.
```

Comment:
Maybe add DataMiner here to be clear on what DLLs we are talking about.
Cause it's still allowed to have references to 3rd party DLLs for example. It's only for DataMiner core DLLs that it should now always be done via devpacks.
 "still contain references to separate **DataMiner** DLL Files"

Reply:
Thanks. Added.